README.md

```diff
@@ -24,13 +24,13 @@ Well, what does that mean?
 Installing OpenNMS
 ==================
 
-For details on installing OpenNMS, please see the install guide: [Installing OpenNMS][].
+For details on installing OpenNMS, please see the installation guide: [Installing OpenNMS][].
```

Comment:
Do we want this to link to the repo (as it does currently) or to the documentation?